packages/buttons/usage.md

```diff
@@ -37,7 +37,7 @@ where this component's less file is imported.
 
 ### Color variables
 
-Color variables referenced in comments are from [cf-core cf-brand-colors.less](https://github.com/cfpb/capital-framework/blob/master/packages/cf-core/src/cf-brand-colors.less).
+Color variables referenced in comments are from [@cfpb/core @cfpb/brand-colors.less](https://github.com/cfpb/design-system/blob/master/packages/core/src/brand-colors.less).
```

Comment:
`@cfpb/brand-colors.less` is correct here and it wouldn't be just brand-colors.less?